### PR TITLE
Add validatorUrl option

### PR DESCRIPTION
**Pull request by mcollina**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by Uzlopak** [comment]
Would prefer to be by default to false and make it a major bump.

**Review by Uzlopak — APPROVED** [review]
I read about this option in the past and its constantly upcoming issues. I totally agree that we should make it configurable. I would prefer to set it to false by default and make it a semver-major.

LGTM

**Comment by mcollina** [comment]
Flipped the default to false, will do a semver-major when landing.

**Review by Uzlopak — APPROVED** [review]

**Review by Eomm — APPROVED** [review]
Ref: https://swagger.io/docs/open-source-tools/swagger-ui/usage/configuration/

File: README.md [hunk 1]
```diff
@@ -305,6 +305,20 @@ await fastify.register(require('@fastify/swagger-ui', {
 
 To ensure that models are correctly rendered at the bottom of the Swagger UI page, it's important to define your schemas using $refs through [fastify.addSchema](https://fastify.dev/docs/latest/Reference/Validation-and-Serialization/#adding-a-shared-schema). Directly embedding JSON schemas within the schema property of your route definitions in Fastify may lead to them not being displayed in Swagger UI.
 
+#### validatorUrl
+
+[SwaggerUI](https://github.com/swagger-api/swagger-ui/) can automatically validate the given sepcification using an online validator.
```
Comment (kibertoad) [inline README.md:310]:
```suggestion
[SwaggerUI](https://github.com/swagger-api/swagger-ui/) can automatically validate the given specification using an online validator.
```

**Comment by Uzlopak** [comment]
@mcollina 
Semver major release please

**Pull request merged**